modules/calib3d/test/test_usac.cpp

```diff
@@ -0,0 +1,105 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+
+namespace opencv_test {
+   TEST(usac_Homography, accuracy) {
+        cv::RNG rng; // random generator
```

Comment:
Use seed from OpenCV test suite (or `cv::theRNG()`).

Totally random numbers must be avoided in OpenCV test - tests must be reproducible.
We must avoid expanding set of tests with sporadic failures.

Reply:
@alalek, in this case the default constructor is used, so the test is deterministic.

But I forgot that before each test OpenCV test system resets "theRNG()" to some fixed state. @ivashmak, if so, please, just put "cv::RNG& rng = cv::theRNG();" and then the rest of the test will remain the same

Reply:
fixed